ngraph/test/visitors/op/round.cpp

```diff
@@ -0,0 +1,42 @@
+// Copyright (C) 2018-2021 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "gtest/gtest.h"
+
+#include "ngraph/ngraph.hpp"
+#include "ngraph/op/util/attr_types.hpp"
+#include "ngraph/opsets/opset1.hpp"
+#include "ngraph/opsets/opset3.hpp"
+#include "ngraph/opsets/opset4.hpp"
+#include "ngraph/opsets/opset5.hpp"
+
+#include "util/visitor.hpp"
+
+using namespace std;
+using namespace ngraph;
+using ngraph::test::NodeBuilder;
+using ngraph::test::ValueMap;
+
+void static test_mode(opset5::Round::RoundMode mode)
+{
+    NodeBuilder::get_ops().register_factory<opset5::Round>();
+    auto data = make_shared<op::Parameter>(element::f32, Shape{200});
+    auto round = make_shared<opset5::Round>(data, opset5::Round::RoundMode::HALF_TO_EVEN);
```

Comment:
@ilyachur  very sorry, I found there is a mistake here, it should be make_shared<opset5::Round>(data, mode); but the code have be merged into master, do I need to create a new PR for it ?